addons/crm_livechat/tests/test_crm_lead.py

```diff
@@ -44,6 +44,21 @@ def test_crm_lead_creation_guest(self):
         self.assertEqual(lead.name, 'TestLead command')
         self.assertEqual(lead.partner_id, self.env['res.partner'])
 
+        # public user: should not be set as customer
+        # note : 'base.public_user' is archived by default
```

Comment:
Maybe add ``self.assertFalse(public_partner.active)`` to ensure test conditions are valid ?